packages/odyssey-react-mui/src/Select.tsx

```diff
@@ -213,7 +213,7 @@ const Select = <
         typeof option === "object"
           ? {
               text: option.text,
-              value: option.value || option.text,
```

Comment:
Curious if this will break any current implementations

Reply:
I thought our examples allowed you to just put the text and no value. Like an array of strings.

Reply:
Yes, `options` has the type of `(string | { text: string, type?: "heading" | "option", value?: string })[]`

Reply:
@bryancunningham-okta the more I look at this, the more I think you may be right. I'll think more about an approach to fix this.

Reply:
@bryancunningham-okta this has been addressed... mind taking another look?

Reply:
Looks good to me!